src/components/inputs/async-select-input/async-select-input.js

```diff
@@ -60,29 +101,19 @@ export class AsyncSelectInput extends React.Component {
         })
       ),
     ]),
-    onChange: PropTypes.func.isRequired,
-    onBlur: PropTypes.func,
     loadOptions: PropTypes.func.isRequired,
-    isMulti: PropTypes.bool,
-    components: PropTypes.object,
-    noOptionsMessage: PropTypes.func,
-    intl: PropTypes.shape({
-      formatMessage: PropTypes.func.isRequired,
-    }).isRequired,
-    hasError: PropTypes.bool,
-    hasWarning: PropTypes.bool,
+    cacheOptions: PropTypes.any,
   };
 
   render() {
     return (
       <Constraints.Horizontal constraint={this.props.horizontalConstraint}>
         <div {...filterDataAttributes(this.props)}>
           <AsyncSelect
-            {...omit(this.props, [
-              'horizontalConstraint',
-              'hasError',
-              'hasWarning',
-            ])}
+            aria-label={this.props['aria-label']}
+            aria-labelledby={this.props['aria-labelledby']}
```

Comment:
maybe just use `filterAriaAttributes` util?

Reply:
There is no point in forwarding additional `aria` attributes to `AsyncSelect` (the one from `react-select`) as it won't do anything with them anyways. This way we're explicit about the ones we support.

Using `filterAriaAttributes` would set the wrong expectation that we're supporting any kind of `aria` props.